book/src/plonk/commitments.md

```diff
@@ -1,60 +0,0 @@
-# Commitments
-
-## What's a commitment
-
-A commitment is a cryptographic primitive that allows you to lock "something" (perhaps a number) without revealing it (yet). This will produce a **commitment**. Later, someone can ask you to reveal the "something". After you do, then can verify that it is linked to the commitment by **opening** it.
-
-<img src="../../img/commitment.png" width="300px">
-
-Commitments are usually defined with these two properties:
-
-* **hiding**: it does not reveal what is commited
-* **binding**: you can't open a commitment to a different value (think second pre-image)
-
-## A simple example
```

Comment:
why remove this section?

Reply:
I was commenting on the simple example section, which talks about SHA-256 :o

Reply:
Ah, because Izzy already had one there.

Reply:
Under "Simple commitments". Pretty comprehensive and clean explanation with hash function.

Reply:
ah indeed! Nice :)